worker/obuilder_build.ml

```diff
@@ -143,3 +146,13 @@ let healthcheck t =
 let cache_stats t =
   let Builder ((module Builder), builder) = t.builder in
   Builder.cache_stats builder
+
+let purge t =
+  let Builder ((module Builder), builder) = t.builder in
+  let before = Unix.gettimeofday () +. prune_margin |> Unix.gmtime in
+  (* set a future time and a big number to ensure everything is deleted *)
+  Builder.prune builder ~before Int.max_int >>= fun n ->
+  Lwt.return n
```

Comment:
`Builder.prune builder ~before Int.max_int : int Lwt.t` and we don't need to operate in the int, here, so we no need to take it of and return it back to the `Lwt.t` context here:

```suggestion
  Builder.prune builder ~before Int.max_int
```